streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -512,4 +630,16 @@ public String newName(final String prefix) {
         return prefix + String.format("%010d", index.getAndIncrement());
     }
 
+    /**
+     * <strong>This function is only for internal usage only and should not be called.</strong>
+     * <p>
+     * Create a unique state store name.
+     *
+     * @param prefix processor name prefix
+     * @return a new unique name
+     */
+    public String newStoreName(final String prefix) {
```

Comment:
Ok, this probably applies to the above function `newName` too since that's what the comments say.